webapp/components/interop.js

```diff
@@ -1,1408 +1,7 @@
-/**
- * Copyright 2022 The WPT Dashboard Project. All rights reserved.
- * Use of this source code is governed by a BSD-style license that can be
- * found in the LICENSE file.
- */
+import { InteropDashboard } from "./interop-dashboard.js";
+import { InteropFeatureChart } from "./interop-feature-chart.js";
+import { InteropSummary } from "./interop-summary.js";
```

Comment:
These lines are causing the lint check to fail because they should be single quotes. Changing them should fix it. 🙂